frontend/ui/data/dictionaries.lua

```diff
@@ -1195,6 +1195,1518 @@ local dictionaries = {
         license = "",
         url = "https://gitlab.com/avsej/dicts-stardict-form-xdxf/raw/d636cc5e8d4a47e22ac7466f4af6d435a8a3f650/002c/stardict-comn_sdict05_brokg-2.4.2.tar.gz"
     },
+    {
+        name = "Adyghe-English Wiktionary",
+        lang_in = "Adyghe",
+        lang_out = "English",
+        entries = 4576,
+        license = "Dual-licensed under CC-BY-SA 3.0 and GFDL",
+        url = "https://github.com/Vuizur/Wiktionary-Dictionaries/raw/master/Adyghe-English%20Wiktionary%20dictionary%20stardict.tar.gz"
```

Comment:
Please add the trailing comma back. Nothing wrong about it; our style asks for it. ;-)